parserator_web/static/js/index.js

```diff
@@ -1,2 +1,58 @@
 /* TODO: Flesh this out to connect the form to the API and render results
    in the #address-results div. */
+
+const form = document.querySelector('form');
+const address = form.elements['address'];
+const btnSubmit = document.querySelector('#submit');
```

Comment:
this doesn't look like it's ever used?

Reply:
I believe form and address are used around line 51 for submitting the form, but you're right about btnSubmit. That was a relic from something else I was trying that I should have cleaned up when I chose to use the form's submit event instead of the submit button's onClick event. Thanks for catching that!